contributing/development/core_and_modules/object_class.rst

```diff
@@ -57,7 +57,7 @@ Registering as virtual is the same but it can't be instanced.
 
 .. code-block:: cpp
 
-    ClassDB::register_virtual_class<MyCustomClass>()
+    ClassDB::register_abstract_class<MyCustomClass>()
```

Comment:
```suggestion
    ClassDB::register_class<MyCustomClass>(true)
```
This is how you register a virtual class, abstract is different, and if we should mention it we should have it talk about abstract classes instead IMO

Reply:
I shouldn't have assume that it only changed the name, my bad.

Reply:
These two should probably be replaced with the better macros:
```cpp
GDREGISTER_CLASS(MyCustomClass);
GDREGISTER_VIRTUAL_CLASS(MyCustomClass);
```